### PR TITLE
Javascript colors

### PR DESCRIPTION
**Pull request by Willhargrave**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -10,4 +10,7 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+@import "components/cards";
```
Comment (moledoPatrik) [inline app/assets/stylesheets/application.scss:13]: import the components inside the index file that is inside components

File: app/assets/stylesheets/components/_colors.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.status-pending {
+  color: orange
+}
```
Comment (moledoPatrik) [inline app/assets/stylesheets/components/_colors.scss:3]: I am not sure about this but looks ok

**Review by moledoPatrik — CHANGES_REQUESTED** [review]

**Review by shoeslace911 — APPROVED** [review]
Colors change!!! wow

**Comment by shoeslace911** [comment]
The colors are too bright. maybe get some ideas from Coolers©

**Review by moledoPatrik — APPROVED** [review]
OK

**Pull request merged**